### PR TITLE
[JENKINS-32493] Adapt to new plugin parent POM

### PR DESCRIPTION
**Pull request by andresrc**
[JENKINS-32493](https://issues.jenkins-ci.org/browse/JENKINS-32493)

@reviewbybees

### CODE REVIEW

**Comment by ghost** [comment]
This pull request originates from a [CloudBees](https://www.cloudbees.com/) employee. At CloudBees, we require that all pull requests be reviewed by other CloudBees employees before we seek to have the change accepted. If you want to learn more about our process please see [this explanation](https://github.com/reviewbybees/about#about-reviewbybees).

File: pom.xml [hunk 1]
```diff
@@ -94,247 +94,8 @@
...
-    <findbugs-maven-plugin.version>2.3.1</findbugs-maven-plugin.version>
-    <gwt-maven-plugin.version>2.1.0-1</gwt-maven-plugin.version>
-    <javancss-maven-plugin.version>2.0</javancss-maven-plugin.version>
-    <jdepend-maven-plugin.version>2.0-beta-2</jdepend-maven-plugin.version>
-    <openjpa-maven-plugin.version>1.2</openjpa-maven-plugin.version>
-    <taglist-maven-plugin.version>2.4</taglist-maven-plugin.version>
-    <versions-maven-plugin.version>1.2</versions-maven-plugin.version>
-    <xml-maven-plugin.version>1.0-beta-3</xml-maven-plugin.version>
+    <jenkins.version>1.580.1</jenkins.version>
+    <java.level>6</java.level>
```
Comment (jtnord) [inline pom.xml:94]: why 1.580.1?

Comment (jglick) [inline pom.xml:94]: Minimum supported by `jenkins-test-harness` 2.0. And just a good all-around cutoff point (SECURITY-144 etc.).

**Comment by jglick** [comment]
:bee:

**Comment by jtnord** [comment]
Still contains a distributionManagent which should be unneeded now

**Comment by andresrc** [comment]
@jtnord thanks!. fixed

**Comment by jtnord** [comment]
:bee:

**Comment by jglick** [comment]
:bee:

**Comment by andresrc** [comment]
@reviewbybees done

**Pull request merged**